server.js

```diff
@@ -1,7 +1,8 @@
 const Hapi = require('hapi');
 const config = require('config');
-
 const routes = require('./routes');
+// This will start creating a connection to database.
+const database = require('./database'); // eslint-disable-line no-unused-vars
```

Comment:
@NarasimhaReddyY This db can be passed inside right ?

Reply:
@prasanna1211 We don't required it to pass inside(it just establish a connection with database while starting server), we'll use mongoose models to all document creation and retrievals from database.

Reply:
@prasanna1211 in `mongoose` you just need to require mongoose and use it once the connection is established. It is not needed hold it in any variable.